core/store/models/job_spec.go

```diff
@@ -234,6 +253,38 @@ func (t Topics) Value() (driver.Value, error) {
 	return string(j), nil
 }
 
+// Feeds holds all flux monitor feed URLs, serializing into the db
+// with ; delimited strings.
+type Feeds []string
```

Comment:
Do you think making this generalized for string arrays would help?

Reply:
Interesting idea. Rule of thumb I have is to DRY things up at 3 repeated uses, but I do worry that this won't be discoverable and will be long forgotten when we do repeat a `;` delimited string array.

One chore that's in the cards is to refactor `Initiator` and `InitiatorParams` to only have the relevant params based on that initiator type. We could address this then.

Going to think some more on this.

Reply:
To address the validation concern we previously discussed, I went ahead and placed the improved URL validation inside the `Feeds` type. This makes it less suitable as a generic string array. But I think we like specific types with it's own validation. Single responsibility, more composable.

Feel free to check out the new validation in `Feeds#UnmarshalJSON`

Reply:
Summary: Did not make a generalized version for the following reasons.

1. Currently only used in one place.
2. We ended up encoding with JSON which is available shared code for others to reuse.
3. We do URI validation on every Feed entry, making it less generic.